WeScan/Scan/ScannerViewController.swift

```diff
@@ -18,35 +24,62 @@ final class ScannerViewController: UIViewController {
     /// The view that draws the detected rectangles.
     private let quadView = QuadrilateralView()
     
+    /// Whether flash is enabled
+    private var flashEnabled = false
+    
+    override var prefersStatusBarHidden: Bool {
+        return true
+    }
+    
     lazy private var shutterButton: ShutterButton = {
         let button = ShutterButton()
         button.translatesAutoresizingMaskIntoConstraints = false
         button.addTarget(self, action: #selector(captureImage(_:)), for: .touchUpInside)
         return button
     }()
     
+    lazy private var cancelButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Cancel", for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(cancelImageScannerController), for: .touchUpInside)
+        return button
+    }()
+    
+    lazy private var toolbar: UIToolbar = {
+        let toolbar = UIToolbar()
+        toolbar.barStyle = .blackTranslucent
+        toolbar.tintColor = .white
+        toolbar.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: 44)
+        return toolbar
+    }()
+    
+    lazy private var autoScanButton: UIBarButtonItem = {
+        return UIBarButtonItem(title: "Auto", style: .plain, target: self, action: #selector(toggleAutoScan))
```

Comment:
We should use localized strings.

Reply:
Localized strings are now used for `Auto`, `Manual`, and `Cancel` on the Scanning screen.